FreeRTOS/FreeRTOS-Kernel
# Use INFINITE_LOOP macro for loop control in unit test

Description
-----------
Use the following pattern for loop control in unit test.
```
for( ; configCONTROL_INFINITE_LOOP(); )
{
    ....
}
```

In the unit test, the configCONTROL_INFINITE_LOOP can be defined to a mocked function for loop controlling.
```
#define configCONTROL_INFINITE_LOOP vFakeInfinitLoop
```

Test Steps
-----------
N/A

Checklist:
----------



Related Issue
-----------
Address #750 comment in line 290.


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
